integrations/ollama/src/haystack_integrations/components/generators/ollama/chat/chat_generator.py

```diff
@@ -197,7 +204,11 @@ def __init__(
             top_p, and others. See the available arguments in
             [Ollama docs](https://github.com/jmorganca/ollama/blob/main/docs/modelfile.md#valid-parameters-and-values).
         :param timeout:
-            Socket timeout *in seconds* for HTTP calls to Ollama.
+            The number of seconds before throwing a timeout error from the Ollama API.
+        :param think
+            If True, the modell will "think" before producing a response.
+            Only [thinking models](https://ollama.com/search?c=thinking) support this feature.
+            The intermediate "thinking" output can be found in the `meta` property of the returned `ChatMessage`.
```

Comment:
currently, linting is failing because this parameter is unused.

I would set `self.think` in `__init__` and then use its value in `run`